src/pages/waiting-room-to-car/waiting-room-to-car.ts

```diff
@@ -291,31 +275,32 @@ export class WaitingRoomToCarPage extends BasePageComponent {
     this.showEyesightFailureConfirmation = show;
   }
 
-  eyesightPassPressed(): void {
-    this.updateForm('eyesightCtrl', 'P');
-    this.store$.dispatch(new EyesightResultPasssed());
-  }
-
-  eyesightFailPressed(): void {
-    this.updateForm('eyesightCtrl', 'F');
-    this.store$.dispatch(new EyesightResultFailed());
-  }
-
   eyesightFailCancelled = () => {
     this.updateForm('eyesightCtrl', null);
     this.store$.dispatch(new EyesightResultReset());
   }
 
-  tellMeQuestionChanged(newTellMeQuestion): void {
+  tellMeQuestionChanged(newTellMeQuestion: TellMeQuestion): void {
     this.store$.dispatch(new TellMeQuestionSelected(newTellMeQuestion));
-    this.form.controls['tellMeQuestionOutcomeCtrl'].setValue('');
+    if (this.form.controls['tellMeQuestionOutcome']) {
+      this.form.controls['tellMeQuestionOutcome'].setValue('');
+    }
   }
 
-  tellMeQuestionCorrect(): void {
-    this.store$.dispatch(new TellMeQuestionCorrect());
+  tellMeQuestionOutcomeChanged(outcome: string): void {
+    if (outcome === 'P') {
+      this.store$.dispatch(new TellMeQuestionCorrect());
+    } else if (outcome === 'DF') {
```

Comment:
No need for the else branch if you return

Reply:
agreed. will change